### PR TITLE
feat(paymaster-example): create example dapp for nft minting with paymaster

### PR DESCRIPTION
**Pull request by jakim929**
- deploys SimpleNFT to use in the example dapp https://optimism-sepolia.blockscout.com/token/0xAB559628B94Fd9748658c46E58a85EfB52FdaCa6
- creates a new vite app (paymaster-nft-mint)
    - example app allows users to generate a modular account and mint an NFT using it 
    - this app currently uses [aa-sdk](https://github.com/alchemyplatform/aa-sdk), in a subsequent PR, will add an example for using [permissionless](https://github.com/pimlicolabs/permissionless.js)
    - the app creates a aa-sdk middleware to use the paymaster
    - I will add a README for this example going through the steps of using the paymaster step by step in a subsequent PR
    
- the example app looks like below

https://github.com/ethereum-optimism/ecosystem/assets/9677071/3714539e-7714-4381-aa2b-2b0d82023d1c

### CODE REVIEW

**Comment by jakim929** [comment]
* **#145** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/ecosystem/145?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @jakim929 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: examples/paymaster-nft-mint/tailwind.config.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+/** @type {import('tailwindcss').Config} */
```
Comment (jakim929) [inline examples/paymaster-nft-mint/tailwind.config.js:1]: copied from ui-components

File: examples/paymaster-nft-mint/src/libraries/aa-sdk/superchainPaymasterMiddleware.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
+import { createSuperchainPaymasterClient } from '@/libraries/aa-sdk/createSuperchainPaymasterClient'
```
Comment (jakim929) [inline examples/paymaster-nft-mint/src/libraries/aa-sdk/superchainPaymasterMiddleware.ts:1]: where the main logic of using the paymaster is. will write a readme about this

File: pnpm-workspace.yaml [hunk 3]
```diff
@@ -1,3 +1,4 @@
 packages:
   - 'apps/*'
   - 'packages/*'
+  - 'examples/*'
```
Comment (nitaliano) [inline pnpm-workspace.yaml:4]: i'm torn if we need an examples workspace or not, what are the negatives if we just put this under apps?

Comment (jakim929) [reply pnpm-workspace.yaml:4]: didn't want to get it confused with apps that we're making for prod use - i would have to subname everything in there as example-paymaster-nft-mint for instance.

but honestly it has almost zero switching costs if we wanna move apps out into the future. down to stick with everything in apps for now

Comment (jakim929) [reply pnpm-workspace.yaml]: cool! updated

**Review by nitaliano — APPROVED** [review]

**Pull request merged**